app/models/project.rb

```diff
@@ -104,6 +104,23 @@ def self.filter_settings
     }
   end
 
+  def self.schema
+    {
+      type: 'object',
+      additionalProperties: false,
+      properties: {
+        id: { type: 'integer', readOnly: true },
+        name: { type: 'string' },
+        description: { type: 'string' },
+        description_html: { type: 'string', readOnly: true },
+        notes: { type: 'object' },
+        creator_id: { type: 'integer', readOnly: true },
+        updater_id: { type: 'integer', readOnly: true },
+        site_ids: { type: 'array', items: { type: 'integer' } }
+      }
+    }.freeze
+  end
```

Comment:
Is it not possible to generate this from the existing project model? If not, default values like `creator_id` should be extracted and standardized across models

Reply:
Good point. https://github.com/QutEcoacoustics/baw-server/issues/480